src/components/Project/hooks/useCountdownClock.ts

```diff
@@ -1,5 +1,5 @@
 import { useEffect, useState } from 'react'
-import { timeSecondsToDateString } from '../utils/timeSecondsToDateString'
+import { timeSecondsToDateString } from '../../../utils/timeSecondsToDateString'
```

Comment:
if its in the root, we can do:

```suggestion
import { timeSecondsToDateString } from 'utils/timeSecondsToDateString'
```

Reply:
Yeah gotcha, this is happening automatically when I move the files around and I'm missing some. I'll keep a better eye on them going forward.